### PR TITLE
Added bogo sort in bash

### PR DESCRIPTION
**Pull request by Ariana1729**
Takes so long to run XD

### CODE REVIEW

File: contents/bogo_sort/code/bash/bogo_sort.bash [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        (( arr[rand] = arr[i] ))
+        (( arr[i] = tmp ))
+    done
+    echo ${arr[*]}
+}
+
+bogo_sort() {
+    local arr
+    arr=("$@")
+    while [ $(is_sorted "${arr[@]}") = 0 ]; do
```
Comment (berquist) [inline contents/bogo_sort/code/bash/bogo_sort.bash]: `[[ ... ]]`, `==` instead of `=`

**Comment by leios** [comment]
Hey @berquist I think this PR looks good. Are we missing anything?

**Comment by berquist** [comment]
Cool, thank you. Hopefully you're learning as much about bash built-ins as I am.

**Comment by Ariana1729** [comment]
Yeah have never ~abused~ used bash like for actual algoithms before this

**Pull request merged**